codespell_lib/data/dictionary.txt

```diff
@@ -1910,6 +1910,7 @@ adjancencies->adjacencies
 adjancency->adjacency
 adjancent->adjacent
 adjancently->adjacently
+adjancey->adjacency
```

Comment:
Perhaps `agency` too ?

Reply:
A quick search of `adjancey` with GitHub and Google search engines currently shows only `adjacency` typos.